frontend/src/pages/distributedWorkloads/global/GlobalDistributedWorkloads.tsx

```diff
@@ -35,31 +36,37 @@ const GlobalDistributedWorkloads: React.FC<GlobalDistributedWorkloadsProps> = ({
       />
     );
   }
-
-  if (activeTab.projectSelectorMode !== null) {
-    if (!namespace && (activeTab.projectSelectorMode === 'singleProjectOnly' || preferredProject)) {
-      // No namespace is in the path and either this tab requires one or we have a preferred one to go to
-      const redirectProject = preferredProject ?? projects[0];
-      return <Navigate to={getInvalidRedirectPath(redirectProject.metadata.name)} replace />;
-    }
-    if (namespace && !projects.find(byName(namespace))) {
-      // This tab allows or requires a namespace but was given an invalid one.
-      return (
-        <ApplicationsPage
-          {...{ title, description }}
-          loaded
-          empty
-          emptyStatePage={
-            <InvalidProject namespace={namespace} getRedirectPath={getInvalidRedirectPath} />
-          }
-        />
-      );
-    }
+  if (!namespace) {
+    const redirectProject = preferredProject ?? projects[0];
```

Comment:
What happens if you have no projects? Can you mount this component?

Reply:
See above `if (projects.length === 0) {`, it renders a "no projects" empty state

Reply:
🤦  yup -- for sure it is...